graphql-dgs-client/src/test/java/com/netflix/graphql/dgs/client/MovieInput.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2021 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.graphql.dgs.client;
+
+import com.netflix.graphql.dgs.client.scalar.DateRange;
+
+import java.util.List;
+
+public class MovieInput {
+    private int movieId;
+    private String title;
+    private Genre genre;
+    private Director director;
+    private List<Actor> actor;
+    private DateRange releaseWindow;
+
```

Comment:
I'd recommend also adding a List<Integer> or some list of non-string but primitive type to the test case.

Reply:
done